src/tests/Microsoft.PowerFx.Interpreter.Tests.Shared/LanguageServiceProtocol/RedesignedLanguageServerTests/Fx2NLTests.cs

```diff
@@ -202,8 +203,43 @@ public async Task TestFx2NLThrows()
             AssertErrorPayload(rawResponse, payload.id, JsonRpcHelper.ErrorCode.InternalError);
             Assert.NotEmpty(TestServer.UnhandledExceptions);
         }
+
+        [Fact]
+        public async Task TestFx2NLWithRequestParamRange()
+        {

```

Comment:
What does this test accomplish? Do we really need to test passing of Range property? There's no functionality being added in this pr. Just adding another property. Imo, there's no point in testing the serialization or its opposite.

Reply:
Yup, thinking more about it we do not necessarily need these tests . But left it to show that user can pass in optional param.